### DISCUSSION TITLE
Suggestion : Multiple Rows for tabs - overcome display bug for tab arrows

### DISCUSSION THREAD

**Discussion by HakMe2Deth** [sumatrapdfreader/sumatrapdf#3015] [Ideas]
Is it possible to have multiple lines of tabs? Not only would this be useful when lots of documents are open, it would also overcome this strange display bug I have where the tab arrows are displayed over the "x" button on the last tab:

![image](https://user-images.githubusercontent.com/106140807/195084419-4d687180-5bef-40da-9fd2-beb2f85dfbe5.png)

Here is the sort of layout I have for Notepad++

![image](https://user-images.githubusercontent.com/106140807/195084936-9c9a00e7-838d-4bb0-8378-bf944f741c90.png)

Thanks for a great little app.  Have used it for many years now and its one of the first things I install on a new PC.
Keep up the good work!

**Comment by GitHubRulesOK** [comment]
requested https://github.com/sumatrapdfreader/sumatrapdf/issues/220
Current status https://github.com/sumatrapdfreader/sumatrapdf/issues/593

**Reply by abelokoj-github** [reply]
I would rather suggest tab scrolling instead. this will keep the interface from being too cluttered. Also, if there is a way to set a fixed width for the file names in opened tabs so that they do not appear too small when the number of tabs are many.

**Another option will be to have an Icon at the top end of the openee tabs, so that you only have a number of tabs showing, and the rest can be shown from a dropdown list when that Icon is clicked.**

**Reply by razorofockham** [reply]
I second the tab scrolling suggestion, great idea!

**Comment by prismriverrr** [comment]
Is this feature still in the plan? It would be great if this feature could be added.
